README.md

```diff
@@ -1467,10 +1467,10 @@ public class MyFirstService : ServiceBase<IMyFirstService>, IMyFirstService
         var waitTask = Task.Run(async () =>
         {
             // ForEachAsync(MoveNext, Current) can receive client streaming.
-            await stream.ForEachAsync(x =>
+            while (await stream.MoveNext())
```

Comment:
Thanks for PR!

Since C# 8 (.NET Core 3.x), you can consume streams in the form of `await foreach` (async-stream) by using `ReadAllAsync` extension method of Grpc.Net.Common.

https://github.com/grpc/grpc-dotnet/blob/530eb1c25c8c37260aaff11cc816adf6a20570f5/src/Grpc.Net.Common/AsyncStreamReaderExtensions.cs#L37

```csharp
await foreach (var x in stream.ReadAllAsync())
```

We will mark `ForEachAsync` helper method as deprecated and recommend using `await foreach`.
Could you please change the sample code to that as well?